knative/deployer.go

```diff
@@ -219,6 +229,19 @@ func updateService(f fn.Function, newEnv []corev1.EnvVar, newEnvFrom []corev1.En
 		for k, v := range f.Annotations {
 			service.ObjectMeta.Annotations[k] = v
 		}
+		// I hate that we have to do this. Users should not see these values.
```

Comment:
Agreed... perhaps these values do not need to be populated and passed through for each Function.  We know that 99.9% of the time they will have the default values.

Perhaps the solution here is to make these health endpoint values default _unless_ they exist in the language pack.  That way we are sticking with the general principle:  most everything is invisible to the Function developer using sane defaults... unless one wants to pop the proverbial hood and override things.

By using defaults here in the Deployer, the Function developer would only see these values populated if the language pack chose to populate them... which would _also_ only need to happen if the language pack developer has a reason to deviate from the default.